### PR TITLE
add .tgr files to highlighting scheme

### PR DESCRIPTION
**Pull request by DwordPtr**
looks like triggers sometimes have .tgr extensions now.

### CODE REVIEW

**Comment by ViViDboarder** [comment]
Thanks for the patch! Where does the `.tgr` extension come from? What tool are you using?

I've noticed the developer console contains extensions of
  * `apxc`: ApexClass
  * `apxt`: ApexTrigger
  * `vfp`: VisualforcePage
  * `vfc`: VisualforceComponent
  * `obj`: Salesforce Object

It might be worth adding these as well.

**Comment by DwordPtr** [comment]
I editing some triggers on sandbox.

I don't exactly know too much about Apex code I've been sort of filing a void on a project. 

I'll take a minute and add those before I go to bed tonight.

This tool has been a real godsend. :)

File: ftdetect/forcedotcom.vim [hunk 1]
```diff
@@ -1,3 +1,4 @@
-autocmd BufNewFile,BufRead *.apxc,*.apxt,*.obj,*.cls,*.trigger,*.tgr set filetype=apex
+autocmd BufNewFile,BufRead *.apxc,*.apxt,*.cls,*.trigger,*.tgr set filetype=apex
 autocmd BufNewFile,BufRead *.vfp,*.vfc,*.page,*.component set filetype=visualforce
 autocmd BufNewFile,BufRead *.log set filetype=apexlog
+autocmd BufNewFile,BufRead *.object,*.obj set filetype=xml
```
Comment (DwordPtr) [inline ftdetect/forcedotcom.vim]: @ViViDboarder  like this?
I haven't ever really messed with vim script before.

Comment (ViViDboarder) [inline ftdetect/forcedotcom.vim]: Looks right to me. Thanks!

**Pull request merged**